scan_filter_chain:
  - name: range_diff_filter
    type: laser_filters/LaserRangeDiffFilter
    params:
      distance_threshold: 1.0
